source/ui/SXRMB/SXRMBHVControlView.h

```diff
@@ -0,0 +1,75 @@
+#ifndef SXRMBHVCONTROLVIEW_H
+#define SXRMBHVCONTROLVIEW_H
+
+#include <QWidget>
+
+#include "beamline/AMControlSet.h"
+#include "beamline/AMControl.h"
+#include "beamline/SXRMB/SXRMBHVControl.h"
+#include "ui/beamline/AMExtendedControlEditor.h"
+
+class QLabel;
+class QPushButton;
+class QVBoxLayout;
+class QDoubleSpinBox;
+
+class SXRMBHVControlView : public QWidget
+{
+    Q_OBJECT
+public:
+	/// the constructor to display a set of HV controls
+	explicit SXRMBHVControlView(AMControlSet * hvControlSet, bool viewOnly = false, QWidget *parent = 0);
+
+protected:
+	/// the HV control set
+	AMControlSet *hvControlSet_;
+
+	///
+};
+
+
+class SXRMBHVControlChannelView : public QWidget
+{
+	Q_OBJECT
+public:
+	/// Constructor for the HV Control Channel View
+	explicit SXRMBHVControlChannelView(SXRMBHVControl * hvControlChannel, bool viewOnly = false, QWidget *parent = 0);
+
+protected slots:
+	/// to handle the HVControlStatusChanged signal
+	void onHVControlStatusChanged(int status);
+	/// to handle the HVControlVoltageValueChanged signal
+	void onHVControlVoltageValueChanged(double value);
+	/// to handle the HVControlCurrentValueChanged signal
+	void onHVControlMeasuredCurrentValueChanged(double value);
+
+	/// to handle the Voltage SpinBox value changed signal
+	void onVoltageSpinBoxEditFinished();
+
+protected:
+	/// to layout the High Voltage control ui components
+	void layoutHVControl(SXRMBHVControl *hvControl, bool viewOnly);
+	/// to setup the signal/slot connections
+	void setupConnections();
+
+protected:
+	/// The HV control
+	SXRMBHVControl *hvControl_;
+
+	/// The layout to hold the ui components
+	QHBoxLayout * contentLayout_;
+	/// the label to display the HV channel name
+	QLabel *nameLabel_;
+	/// the label to display the Voltage value, which is only visible for viewOnly
+	QLabel *actVoltageLabel_;
```

Comment:
@XuetaoLiu-CLS, you don't need to use abbreviations for memer names.  You don't need to change this, just sending a reminder that full names are always better! :)